server.go

```diff
@@ -6,22 +6,29 @@ import (
 	"net/http"
 	"strconv"
 
+	"github.com/DataDog/datadog-go/v5/statsd"
 	"github.com/gorilla/mux"
+
+	log "github.com/sirupsen/logrus"
 )
 
 type ToDoServer struct {
 	ToDoStore *ToDoStore
+	Statsd    *statsd.Client
 }
 
-func NewServer() *ToDoServer {
+func NewServer(statsd *statsd.Client) *ToDoServer {
 	return &ToDoServer{
 		ToDoStore: NewToDoStore(),
+		Statsd:    statsd,
 	}
 }
 
 func (server *ToDoServer) GetToDos(w http.ResponseWriter, req *http.Request) {
 	j, _ := json.Marshal(server.ToDoStore.Tasks)
 	fmt.Fprintf(w, string(j))
+	server.Statsd.Count("todo_tasks_uncompleted.count", int64(server.ToDoStore.Uncompleted), []string{"environment:dev"}, 1)
```

Comment:
This metric would be clearer as a Gauge. The reason is that count metrics are intended to track things as they occur and then the counts get accumulated. 
That doesn't apply here, because what if there are two `GET` requests? That would result in two `count` dogstatsd messages getting sent, which would then get added together, so it would look like there are 2x uncompleted tasks.
If you send this a gauge, then its "last writer wins" behavior and there would be no double-counting.